hardware_interface/README.md

```diff
@@ -0,0 +1,299 @@
+# Hardware Interfaces           {#mainpage}
+
+hardware interfaces are used by ROS control in conjunction with one of the 
+available ROS controllers to send (\ref hardware_interface::RobotHW::write) 
+commands to the hardware and receive (\ref hardware_interface::RobotHW::read) 
+states from the robot's resources (joints, sensors, actuators).
+
+A list of available hardware interfaces (provided via the HardwareResourceManager) 
+as of this writing:
+
+- [JointCommandInterface](\ref hardware_interface::JointCommandInterface): 
```

Comment:
Not sure if these links will work in the README.md when viewed on the GitHub repo. It is working though in the generated doxygen documentation.

Reply:
Looks like they do not work :(

https://github.com/fjp/ros_control/tree/melodic-devel/hardware_interface